edweiss-app/__test__/home_page/getCurrentDay.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { getCurrentDay } from '@/components/core/getCurrentDay';
+
+describe('getCurrentDay', () => {
+    it('should return the current day of the week as a number', () => {
```

Comment:
Thank you for your feedback! I agree that testing the current time is quite challenging due to its dynamic nature. Given this complexity, I think it might be best to remove this test altogether to avoid unreliable results.

Let me know if you have any other suggestions!